### DISCUSSION TITLE
0.6.8 beta-3

### DISCUSSION THREAD

**Discussion by mdelobelle** [mdelobelle/metadatamenu#477] [Announcements]
# Metadata Menu 0.6.8 (beta)

## Improvement
- fileclass view reopening at start
- code block for fileclass table view
- existing saved view can now be modified
- choose fileclass at filecreation
- add fileclass to file command

## Fix
- formula not updating on click
- single char field not parsed

<hr /><em>This discussion was created from the release <a href='https://github.com/mdelobelle/metadatamenu/releases/tag/0.6.8-beta-3'>0.6.8 beta-3</a>.</em>

**Comment by jayambede** [comment]
Improvement:

- metadata field buttons in File Properties pane ... 

... or is that not possible/highly complicated to achieve?
